docs/search/intellisense.md

```diff
@@ -0,0 +1,78 @@
+
+From the [**New UI**](/docs/get-started/sumo-logic-ui), go to the left nav and click **Logs** > **Log Search**.  
+
+### 2. Use Syntax autocompletion
+
+1. Clicking in the empty query field to generate a list of suggested metadata field entries to start your search.<br/><img src={useBaseUrl('img/search/intellisense/more-information.png')} alt="more-information.png" style={{border: '1px solid gray'}} />
+1. Type a search operator like `parse` to begin a query clause, and Intellisense will suggest options. Refer to the [Search Operator](/docs/search/search-query-language/search-operators) guide for syntax and examples.
+1. Use your keyboard's **arrow keys** or cursor to scroll through the suggestions. Press **Enter** or click to select an entry.
+1. Depending on the search operators you're including, you may need to use pipes (`|`) to separate query segments. For example, the `where` operator must be separated from other operators by the pipe symbol (`|`).
+
+:::tip
+To learn more about a suggestion before adding it to your query, click the "i" icon. This shows you more information, such as the field type (e.g., metadata, keyword), definition, examples, and/or link to doc.<br/><img src={useBaseUrl('img/search/intellisense/initial-prompts.png')} alt="initial-prompts.png" style={{border: '1px solid gray'}} />
+:::
+
+### 3. Key-value pair support
+
+1. For JSON or structured logs, type a key (e.g., `status`). Select the suggested facet (e.g., `timestamp`) for more precise filtering.
+1. Intellisense will suggest keys and values. Select the appropriate one using **Enter**.
+
+### 4. Next-operator prediction
+
+1. Add a filtering clause like `where status="error"` and group results using `group by`.
+1. Intellisense will predict and suggest the next operator (e.g., `group by`, `where`). Use **arrow keys** or click to select.
+
+### 5. Revamped log assist UI
+
+1. As you write, the UI will highlight syntax issues. Hover over the underlined errors for suggested fixes.
+1. The side panel provides query-building tutorials and examples for additional support.
+
+### 6. Facet determination for JSON logs
+
+1. When querying for JSON-formatted logs, Intellisense will suggest facets like `status` or `timestamp`.
+1. Click the facet to refine your query.
+
+By following these steps, you can take full advantage of Intellisense to accelerate query writing and improve search efficiency.
+
+
+## Leverage Copilot AI
+
+In a hurry to complete your investigation? Need help writing queries?
+
+Try [Sumo Logic Copilot](/docs/search/copilot), our AI-powered assistant that lets you enter natural questions in plain English and auto-generate queries - no need to write them manually. It guides you through query building step-by-step with AI-driven suggestions to refine your results. Because all your observability and security source records are in Sumo Logic, Copilot helps discover related datasets.
```

Comment:
```suggestion
Still need help writing queries? Try [Sumo Logic Copilot](/docs/search/copilot), our AI-powered assistant that lets you enter natural questions in plain English and auto-generate queries - no need to write them manually. It guides you through query building step-by-step with AI-driven suggestions to refine your results. Because all your observability and security source records are in Sumo Logic, Copilot helps discover related datasets.
```